### PR TITLE
Add link preview support in WAC

### PR DESCRIPTION
**Pull request by Robi9**
Support to include a URL preview in messages with links sent in Whatsapp. Add `"preview_url": true` to the Message Text Object as indicated in the [documentation](https://developers.facebook.com/docs/whatsapp/cloud-api/reference/messages#:~:text=of%20type%20video.-,Text%20Object,-Name).

### CODE REVIEW

File: handlers/facebookapp/facebookapp.go [hunk 1]
```diff
@@ -982,7 +982,8 @@ type wacComponent struct {
 }
 
 type wacText struct {
-	Body string `json:"body"`
+	Body       string `json:"body"`
+	PreviewURL bool   `json:"preview_url,omitempty"`
```
Comment (norkans7) [inline handlers/facebookapp/facebookapp.go:986]: Why not always have that explicitly set instead of omitting that

Comment (Robi9) [reply handlers/facebookapp/facebookapp.go:986]: If it is not sent it will be false by default so I thought of omitting it when I don't need to send it.

Comment (Robi9) [reply handlers/facebookapp/facebookapp.go:986]: Do you think it's better to make it explicit?

File: handlers/facebookapp/facebookapp.go [hunk 2]
```diff
@@ -1086,8 +1086,14 @@ func (h *handler) sendCloudAPIWhatsappMsg(ctx context.Context, msg courier.Msg)
 			} else {
 				if i < (len(msgParts) + len(msg.Attachments()) - 1) {
 					// this is still a msg part
+					text := &wacText{}
```
Comment (norkans7) [inline handlers/facebookapp/facebookapp.go:1089]: text := &wacText{PreviewURL: false}

**Review by norkans7 — APPROVED** [review]

**Pull request merged**